.github/scripts/filter_changed_files.sh

```diff
@@ -1,7 +1,4 @@
 #!/bin/bash
-# Copyright (c) HashiCorp, Inc.
```

Comment:
[question][non-blocker] any reason this was removed?

Reply:
No, not sure what happened there. I'll try to look into it.

Reply:
There were some merge issues we cleaned up so this should now be ok. Resolving